db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
Well, you can't assume the sensors to be present. If a sensor is missing that's a good test case.